hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -83,6 +84,22 @@ private OMConfigKeys() {
       "ozone.key.deleting.limit.per.task";
   public static final int OZONE_KEY_DELETING_LIMIT_PER_TASK_DEFAULT = 20000;
 
+  public static final String OZONE_OM_OPEN_KEY_CLEANUP_SERVICE_INTERVAL =
+      "ozone.om.open.key.cleanup.service.interval";
+  public static final Duration
+      OZONE_OM_OPEN_KEY_CLEANUP_SERVICE_INTERVAL_DEFAULT =
+      Duration.ofHours(24);
+
+  public static final String OZONE_OM_OPEN_KEY_EXPIRE_THRESHOLD =
+      "ozone.om.open.key.expire.threshold";
+  public static final Duration OZONE_OM_OPEN_KEY_EXPIRE_THRESHOLD_DEFAULT =
```

Comment:
Every other Ratis `TimeDuration` in this class is related to Ratis.
Either it starts with `ozone.om.ratis.server` or `ozone.om.snapshot`.
I changed this to java `Duration` because I think this config is not related to Ratis.
Do you think we still need to keep it as `TimeDuration`?

Reply:
Yes, String is indeed better. And it's also the same representation as in `ozone-default.xml`.